benchmarking/nursery/benchmark_automl/README.md

```diff
@@ -0,0 +1,42 @@
+# Benchmark Automl
```

Comment:
Benchmark Automl submission? Just Automl is a bit confusing

Reply:
Completely agree, will reword.